bats/tests/preferences/verify-paths.bats

```diff
@@ -0,0 +1,97 @@
+# Test case 30
+
+setup() {
+    load '../helpers/load'
+    if is_windows; then
+        skip "test not applicable on Windows"
+    fi
+    # Ensure subshells don't inherit a path that includes ~/.rd/bin
+    export PATH=$(echo "$PATH" | tr ':' '\n' | grep -v /.rd/bin | tr '\n' ':')
+}
+
+teardown_file() {
+    load '../helpers/load'
+    run rdctl shutdown
+    assert_nothing
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start app and fix dotfiles' {
+    start_container_engine
+    wait_for_container_engine
+    ensure_dotfiles_are_completed_BUG_BUG_BUG_4519
+}
+
+# Running `bash -l -c` causes bats to hang
+@test 'bash managed' {
+    if command -v bash >/dev/null; then
+        run bash -l -c "which rdctl" 3>&-
+        assert_output --partial "$HOME/.rd/bin/rdctl"
+    else
+        skip 'bash not found or ~/.bashrc does not exist'
```

Comment:
You are not testing if `~/.bashrc` exists or not

```suggestion
        skip 'bash not found'
```

Reply:
This seems to be still unaddressed

Reply:
Right, missed that